webpack.dev.js

```diff
@@ -21,7 +21,7 @@ module.exports = merge(common, {
     },
     port: 3001,
     // eslint-disable-next-line no-shadow
-    writeToDisk: filePath => /index\.html/.test(filePath),
+    writeToDisk: true,
```

Comment:
This change should allow for navigation to both `localhost:3000` and `localhost:3001`